source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -858,11 +893,13 @@ JSON Schema. This operation creates a document similar to the following:
 Query for Documents on a Deterministically Encrypted Field
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Queries on encrypted fields can be done using traditional MongoDB driver
-methods as well. When a query is made using a client configured to use
-CSFLE, such as when a doctor in the MedcoMD management system searches
-for a patient with their SSN, the patient's data is returned
-unencrypted:
+.. image:: /figures/CSFLE_Read_Encrypted.png
+   :alt: Diagram that shows the data flow for reading field-level encrypted data
+
+Queries on documents with encrypted fields can be run using conventional
+MongoDB driver methods. When a doctor performs a query in the MedcoMD
+management system to search for a patient by their SSN, the patient's data
+is returned unencrypted:
```

Comment:
nit: Isn't this really that "the driver unencrypts the patient's data before returning it"?

Reply:
Thanks, will update.